lib/va_profile/v2/contact_information/service.rb

```diff
@@ -306,8 +310,12 @@ def post_or_put_data(method, model, path, response_class)
             # in_json_v2 method should replace in_json after Contact Information V1 has depreciated
             if path == 'addresses' && log_transaction_id?
               Rails.logger.info("ContactInformationV2 METHOD: #{method}, POST OR PUT JSON: #{model.in_json_v2},
-                ADDRESS POU: #{model.address_pou}, REQUEST PATH: #{request_path},
-                CREATED AT: #{model.created_at}, UPDATED AT: #{model.updated_at}")
+                ADDRESS POU: #{model.address_pou}, REQUEST PATH: #{request_path}")
+              if Flipper.enabled?(:override_address_pou) && model.address_pou == ('RESIDENCE' || 'RESIDENCE/CHOICE')
+                # OVEERRIDE ADDRESS_POU
+                model.address_pou = 'RESIDENCE/CHOICE'
+                Rails.logger.info("ContactInformationV2 OVERRIDE ADDRESS POU JSON: #{model.in_json_v2}")
+              end
```

Comment:
Log to ensure the address_pou is overridden and address_pou is 'RESIDENCE/CHOICE'